src/test/resources/hudson/tasks/configuration-as-code.yaml

```diff
@@ -0,0 +1,11 @@
+tool:
+  ant:
+    installations:
+      - name: "Ant 1.10.5"
```

Comment:
Will it really work without home directory?

Reply:
yes because of the fix in Ant#AntInstallation class (https://github.com/jenkinsci/ant-plugin/pull/36/files#diff-f1d43b50081dbc46a52cfedbf7b8e710R359)